01_basic/README.md

```diff
@@ -19,26 +19,58 @@ Application properties can be initialized from environment variables before runn
 
 ## Installation
 
+To build the workflow image and push it to the image registry, use the [./scripts/build.sh](../scripts/build.sh) script:
+```bash
+This script performs the following tasks in this specific order:
+1. Generates a list of Operator manifests for a SonataFlow project using the kn-workflow plugin (requires at least v1.35.0)
+2. Builds the workflow image using podman or docker
+3. Optionally, deploys the application:
+    - Pushes the workflow image to the container registry specified by the image path
+    - Applies the generated manifests using kubectl in the current k8s namespace
+
+Usage: 
+    ./scripts/build.sh [flags]
+
+Flags:
+    -i|--image=<string> (required)       The full container image path to use for the workflow, e.g: quay.io/orchestrator/demo.
+    -b|--builder-image=<string>          Overrides the image to use for building the workflow image.
+    -r|--runtime-image=<string>          Overrides the image to use for running the workflow.
+    -n|--namespace=<string>              The target namespace where the manifests will be applied. Default: current namespace.
+    -m|--manifests-directory=<string>    The operator manifests will be generated inside the specified directory. Default: 'manifests' directory in the current directory.
+    -w|--workflow-directory=<string>     Path to the directory containing the workflow's files (the 'src' directory). Default: current directory.
+    -P|--no-persistence                  Skips adding persistence configuration to the sonataflow CR.
+       --deploy                          Deploys the application.
+    -h|--help                            Prints this help message.
+
+Notes: 
+    - This script respects the 'QUARKUS_EXTENSIONS' and 'MAVEN_ARGS_APPEND' environment variables.
+```
+
 Use the scripts:
-* Build and push the image:
+1. Build the image and generate the manifests:
 ```
-cd ..
-WORKFLOW_ID=basic WORKFLOW_FOLDER=01_basic WORKFLOW_IMAGE_REGISTRY=quay.io WORKFLOW_IMAGE_NAMESPACE=orchestrator ./scripts/build-push.sh
+../scripts/build.sh --image=quay.io/orchestrator/demo-basic
 ```
-* Generate manifests that have to be applied on the OCP cluster wiht RHDH and OSL:
+
+The manifests location will be displayed by the script.
+2. Push the image
 ```
-WORKFLOW_ID=basic WORKFLOW_FOLDER=01_basic WORKFLOW_IMAGE_REGISTRY=quay.io WORKFLOW_IMAGE_NAMESPACE=orchestrator ./scripts/gen-manifest.sh
+docker push <image>
```

Comment:
podman?

Reply:
should be `pocker` x))
I will set `pocker` and use it